ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/mac-security-eth-po-entropy.yml

```diff
@@ -33,7 +33,8 @@ mac_security:
 
 ### Mgmt sec
 management_security:
-  entropy_source: hardware
+  entropy_source:
+    hardware: true
```

Comment:
This change is breaking for existing deployments. I think we need to add a new key with your new model like `entropy_sources` in parallel with the old, and then deprecate the old. You will need to read both new and old in the templates. (Yes it may lead to duplicate config if someone uses both).

Reply:
Fixed